homeassistant/components/utility_meter/sensor.py

```diff
@@ -400,6 +398,9 @@ def extra_state_attributes(self):
             state_attr[ATTR_CRON_PATTERN] = self._cron_pattern
         if self._tariff is not None:
             state_attr[ATTR_TARIFF] = self._tariff
+        if last_reset := self._last_reset:
```

Comment:
```suggestion
        # last_reset in utility meter was created before last_reset
        # for long term statistics in base sensor.
        # To avoid a breaking change we set last_reset directly
        # in extra state attributes.
        if last_reset := self._last_reset:
```